src/openzaak/templates/admin/object_history.html

```diff
@@ -0,0 +1,117 @@
+                                        <h4>{% trans 'Oud' %}</h4>
+                                    </div>
+                                    <div class="col">
+                                        <h4>{% trans 'Nieuw' %}</h4>
+                                    </div>
+                                </div>
+                                {% for change in changes %}
+                                    {% if change.0 == 'add' %}
+                                        {% for field, diff in change.1.items %}
+                                            <div class="row">
+                                                <div class="col"><code>{{ field }}</code></div>
+                                                <div class="col"></div>
+                                                <div class="col"><p>{{ diff }}</p></div>
+                                            </div>
+                                            <hr/>
+                                        {% endfor %}
+                                    {% elif change.0 == 'change' %}
+                                        {% for field, diff in change.1.items %}
+                                            <div class="row">
+                                                <div class="col"><code>{{ field }}</code></div>
+                                                <div class="col">{{ diff.0 }}</div>
+                                                <div class="col lg-2">{{ diff.1 }}</div>
+                                            </div>
+                                            <hr/>
+                                        {% endfor %}
+                                    {% elif change.0 == 'remove' %}
+                                        {% for field, diff in change.1.items %}
+                                            <div class="row">
+                                                <div class="col"><code>{{ field }}</code></div>
+                                                <div class="col">{{ diff }}</div>
+                                                <div class="col lg-2"></div>
+                                            </div>
+                                            <hr/>
+                                        {% endfor %}
+                                    {% endif %}
+                                {% endfor %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        {% endfor %}
+        </tbody>
+    </table>
+{% endif %}
+</div>
+</div>
+
+<link rel="stylesheet" href="{% static 'vng_api_common/libs/bootstrap/css/bootstrap.min.css' %}">
+<script src="{% static 'vng_api_common/libs/jquery/jquery-3.3.1.slim.min.js' %}"></script>
+<script src="{% static 'vng_api_common/libs/popper/popper.min.js' %}"></script>
+<script src="{% static 'vng_api_common/libs/bootstrap/js/bootstrap.min.js' %}"></script>
```

Comment:
Not sure these scripts and styles are needed?

Reply:
One wasn't needed indeed, but the other three are needed for the modal to show the changes